main/EDMainMod.F90

```diff
@@ -270,7 +271,9 @@ subroutine ed_integrate_state_variables(currentSite, bc_in )
     real(r8) :: dbh_old               ! dbh of plant before daily PRT [cm]
     real(r8) :: hite_old              ! height of plant before daily PRT [m]
     real(r8) :: leaf_c
-    
+    real(r8) :: delta_dbh             ! correction for dbh
```

Comment:
should we do anything with the info in delta_dbh and delta_hite, (i.e. send one of them to a history variable) or is that overkill?

Reply:
I just left them there in case someone wanted to use them for diagnostics in the future, as a convenience.  I have no plans of adding them to the history file personally.

Reply:
ok, works for me.  it's nice to have that there in case we ever suspect something weird is going on with the structure resets.